desktop/core/src/desktop/js/onePageViewModel.js

```diff
@@ -182,14 +182,12 @@ class OnePageViewModel {
       $.ajax({
         url: scriptUrl,
         converters: {
-          'text script': function (text) {
-            return text;
-          }
+          'text script': text => text
```

Comment:
Just out of curiosity, was this change needed or just an improvement?

Reply:
this was not necessary, just an improvement.